src/local-time.lisp

```diff
@@ -188,6 +191,30 @@
 ;; time of day.
 (defparameter +modified-julian-date-offset+ -51604)
 
+(defun %subzone-as-of (timezone &key unix-time timestamp)
```

Comment:
Avoid `&key` usage here and just use `unix-time`. Timestamp conversion can be done by the caller if needed.

Reply:
Yep, done